config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -60,6 +60,8 @@
   <!-- test resources are weird by design, no validations in them -->
   <suppress id="lineLength"
          files="src[\\/]it[\\/]resources[\\/].*[\\/]InputLineLength.*\.java"/>
+  <suppress id="lineLength"
+            files="src[\\/]it[\\/]resources[\\/].*[\\/]InputMethodsAndConstructorsAnnotations.*\.java"/>
```

Comment:
this is not ok,
please wrap all code in Inputs to not be lengthy.

Reply:
done. removed all these suppressions. Wrapped code:

https://github.com/checkstyle/checkstyle/blob/b0dd2d65f8f163adef2fb6f9d2e6cc21152390c5/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4853methodsandconstructorsannotations/InputMethodsAndConstructorsAnnotations.java#L78-L79